src/spawn_stubs.c

```diff
@@ -648,22 +679,22 @@ CAMLprim value spawn_windows(value v_env,
   if (!dup2_and_clear_close_on_exec(v_stdin , &si.hStdInput ) ||
       !dup2_and_clear_close_on_exec(v_stdout, &si.hStdOutput) ||
       !dup2_and_clear_close_on_exec(v_stderr, &si.hStdError )) {
-    win32_maperr(GetLastError());
+    caml_win32_maperr(GetLastError());
     close_std_handles(&si);
     uerror("DuplicateHandle", Nothing);
```

Comment:
If we error out here, we will leak the strings allocated above. Maybe invert the order?

Reply:
Oh blimey, good catch. (Another) revised version incoming...

Reply:
Done!